spyder/widgets/comboboxes.py

```diff
@@ -164,11 +164,11 @@ class PatternComboBox(BaseComboBox):
     """Search pattern combo box"""
 
     def __init__(self, parent, items=None, tip=None,
```

Comment:
```suggestion
    def __init__(
        self,
        parent,
        items=None,
        tip=None,
```

Fix formatting (I have to leave two suggestions because Github doesn't allow me to do it in one).